### ISSUE TITLE
ESLint warning

### ISSUE THREAD

**Issue by alexbruno** [SoftboxLab/gandalf-lint#5]
Last ESLint updates changed some settings and now we have this warning related to Standard config:

(node:5933) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "node_modules/eslint-config-standard/index.js")

Maybe a dependencies update will solve it, community is already tracking this warning status:

https://github.com/standard/eslint-config-standard-react/issues/48

**Comment by pablohpsilva** [comment]
I'll be on it ASAP. Thank you!

**Comment by sheerun** [comment]
@pablohpsilva Thank you for handling it, it's long-standing issue for me

**Comment by pablohpsilva** [comment]
Hello Guys. 
@sheerun I just fixed it but I did something more that was on our milestone: removed the vue + react linters. They will be moved to another package, respectively: [gandalf-lint-vue](https://github.com/SoftboxLab/gandalf-lint-vue) and [gandalf-lint-react](https://github.com/SoftboxLab/gandalf-lint-react). This change was mentioned on issue #3 .

All the dependencies were updated and can be found on PR #6 

Thank you guys! Just waiting on PR acceptance by @alexbruno and @victormaestri :)
